src/main/java/com/cloudbees/jenkins/support/configfiles/OtherConfigFilesComponent.java

```diff
@@ -0,0 +1,76 @@
+package com.cloudbees.jenkins.support.configfiles;
+
+import com.cloudbees.jenkins.support.api.Component;
+import com.cloudbees.jenkins.support.api.Container;
+import com.cloudbees.jenkins.support.api.FileContent;
+import com.cloudbees.jenkins.support.util.Helper;
+import edu.umd.cs.findbugs.annotations.NonNull;
+import hudson.Extension;
+import hudson.security.Permission;
+import jenkins.model.Jenkins;
+import org.xml.sax.SAXException;
+
+import javax.xml.parsers.ParserConfigurationException;
+import javax.xml.stream.XMLStreamException;
+import javax.xml.transform.TransformerException;
+import java.io.File;
+import java.io.FilenameFilter;
+import java.io.IOException;
+import java.util.Collections;
+import java.util.Set;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * Support component for adding xml files to the support bundle.
+ */
+@Extension
+public class OtherConfigFilesComponent extends Component {
+    @NonNull
+    @Override
+    public Set<Permission> getRequiredPermissions() {
+        return Collections.singleton(Jenkins.ADMINISTER);
+    }
+
+    @NonNull
+    @Override
+    public String getDisplayName() {
+        return "Other Jenkins Configuration Files (secrets are filtered out unless saved in clear in the .xml files)";
```

Comment:
Text is slightly better but I still think it is better to focus on the positive, e.g. We redact the encrypted secrets... which leaves unsaid but implies the non-redaction of "secrets" that are not redacted